### PR TITLE
91_version_history file updates for 2.1.0 release

### PR DESCRIPTION
**Pull request by Mohit04tomar**

### CODE REVIEW

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
        - The asset browser APIs have been enhanced to allow for a window of data in the past to be returned. In conjunction a new timespan entry point has been added to allow the oldest and newest date for which an asset exists within the reading buffer to be returned.
        - An option has been added to the advanced configuration of south services that allow the statistics that are generated by the south service to be tailored. Statistics may be kept for the service as a whole, each asset ingested by the service or both. This setting relates to a given service and may be different in different south services. Full details are available in the tuning guide within the documentation.
+       - Two new types of user are now available in Fledge; users that can view the configuration only and users that can view the data only.
+       - The OMF north plugin now supports all ADH regions.
+       - An issue that could cause properties of configuration items to be lost or incorrectly updated has been resolved.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:43]: This is a bug fix?

File: docs/91_version_history.rst [hunk 2]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
 
 
-    - Bug Fix:
-
-       - The reset and purge scripts have been improved such that if the readings plugin is different from the storage plugin the data will be removed from the appropriate plugins.
+    - Bug Fixes
+      
+        - The reset and purge scripts have been improved such that if the reading plugin is different from the storage plugin the data will be removed from the appropriate plugins.
        - A problem that prevented items from being disabled in the user interface when they were not valid for the current configuration has been resolved.
        - An issue that would sometimes cause the error \Not all updates in a transaction succeeded\ to be logged when updating the users access token has been resolved.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:50]: Use ticks for rst code syntax of instead of \\

File: docs/91_version_history.rst [hunk 3]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
 
 
 - **GUI**
 
-    - New Features:
+    - Features
 
+       - The graphical user interface for viewing the configuration of the south and north services and tasks has now been updated to display the configuration items in multiple tabs.
+       - The user interface now supports two types of view only users; those that can view the configuration and those that can view the data only.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:58]: Next item is not the same? Do we have this in core section?

File: docs/91_version_history.rst [hunk 4]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
+       - A new mechanism has been introduced that allows configuration items within a category to have a group associated with them. This allows items that relate to particular mechanism be recognised as related by clients of the API and display decisions to be taken based on these groups.
        - The asset browser APIs have been enhanced to allow for a window of data in the past to be returned. In conjunction a new timespan entry point has been added to allow the oldest and newest date for which an asset exists within the reading buffer to be returned.
        - An option has been added to the advanced configuration of south services that allow the statistics that are generated by the south service to be tailored. Statistics may be kept for the service as a whole, each asset ingested by the service or both. This setting relates to a given service and may be different in different south services. Full details are available in the tuning guide within the documentation.
+       - Two new types of user are now available in Fledge; users that can view the configuration only and users that can view the data only.
+       - The OMF north plugin now supports all ADH regions.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:42]: Move to plugins

File: docs/91_version_history.rst [hunk 5]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
+       - The OMF north plugin has been updated to allow support for OMF 1.2 features. This allows for better control of types within OMF resulting in the OMF plugin now dealign more cleanly with assets with different datapoints in different readings. Any assets that are already being sent to an OMF endpoint will continue to use the previous type mechanism. A number of new OMF hints are also supported.
+       - The S2OPCUA south plugin has been updated to resolve an issue with duplicate browse names causing data from two OPC UA variables being stored in the same Fledge datapoint. The plugin has also been updated to give more options for how the assets are structured. The option of a single asset for all datapoints and an asset put OPC UA object have been added. It is also possible to use the OPC UA object name as the prefix for asset names in the case of a single variable per asset as well as the current option of a fix prefix for the browse name of the variable.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:74]: Move to bug fixes

File: docs/91_version_history.rst [hunk 6]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
+       - The S2OPCUA south plugin has been updated to allow the timestamp for readings to be taken from the OPC UA server itself rather than the time that it was received by Fledge.
 
 
-    - Bug Fix:
+    - Bug Fixes
 
+       - A number of bug fixes have been made to the North OPC UA Client plugin to improve the logging and responsiveness of the plugin.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:80]: Fine; though packages are being released first time?

File: docs/91_version_history.rst [hunk 7]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
 
 - **Fledge Core**
 
-    - New Features:
+    - New Features
 
+       - North plugins run as a task rather than a service would be run by the Python sending task rather than the C++ sending task. This resulted in filter pipelines not being applied to the task. This has now been resolved.
+       - A new mechanism has been introduced that allows configuration items within a category to have a group associated with them. This allows items that relate to particular mechanism be recognised as related by clients of the API and display decisions to be taken based on these groups.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:38]: ... that relate to a particular ...

File: docs/91_version_history.rst [hunk 8]
```diff
@@ -32,44 +32,56 @@ Release Date: 2022-12-26
...
+      
+        - The reset and purge scripts have been improved such that if the reading plugin is different from the storage plugin the data will be removed from the appropriate plugins.
        - A problem that prevented items from being disabled in the user interface when they were not valid for the current configuration has been resolved.
        - An issue that would sometimes cause the error \Not all updates in a transaction succeeded\ to be logged when updating the users access token has been resolved.
 
 
 - **GUI**
 
-    - New Features:
+    - Features
```
Comment (praveen-garg) [inline docs/91_version_history.rst:55]: Keep New as with Core or remove from there as well

**Review by praveen-garg — CHANGES_REQUESTED** [review]

File: docs/91_version_history.rst (hunk 6 above)
Comment (Mohit04tomar) [reply docs/91_version_history.rst:80]: So, this sanitized description is not required and should be removed.

**Review by praveen-garg — APPROVED** [review]

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**